### PR TITLE
Testrun

### PR DESCRIPTION
**Pull request by SylivanKenobi**
The Techlab flows a lot better now. The use of one Application is a good idea. I added comments everywhere where I had specific issues. 
Issues overall:
* There are Labs who edit templates who aren't created because they are from github.
* The Labs at the end get shorter and shorter. They don't feel complete yet.
* Most labs would profit from a bit more explanation on why we add for example a pipeline.

### CODE REVIEW

File: content/en/docs/05.0/appmonitoring.md [hunk 1]
```diff
@@ -115,7 +114,8 @@ Open Grafana (URL provided by the trainer) and switch to the explore tab, then e
 Make sure to replace `<namespace>` with your current namespace
 {{% /alert %}}
 
-
+<!-- Didn't work had no data showing -->
```
Comment (phil-pona) [inline content/en/docs/05.0/appmonitoring.md:117]: Which user did you use?

File: content/en/docs/04.0/pipeline.md [hunk 2]
```diff
@@ -369,14 +369,14 @@ Moreover, the logs can be viewed with the following command and selecting the ap
...
 
-
 ## Task {{% param sectionnumber %}}.6: OpenShift WebUI
 
 With the OpenShift Pipeline operator, a new menu item is introduced to the WebUI of OpenShift. All Tekton CLI commands, which are used above, could be replaced with the web interface. The big advantage is the graphical presentation of Pipelines and their lifetime.
 
 
 ### Checking your application
-
+<!-- This lab didn't work. The Pipeline was succesful but the application didn't start. The pods searched for kafka which didn't exist -->
```
Comment (phil-pona) [inline content/en/docs/04.0/pipeline.md:378]: I changed it to the rest branch, the pods should be able to start now.

File: content/en/docs/05.0/appmonitoring.md (hunk 1 above)
Comment (SylivanKenobi) [reply content/en/docs/05.0/appmonitoring.md:117]: Hannelore14

Comment (phil-pona) [reply content/en/docs/05.0/appmonitoring.md:117]: it's working again, prometheus was down

**Comment by chrira** [comment]
Integrated without merging, thanks.

**Pull request closed without merging**